### PR TITLE
Decrease the block time threshold (2x)

### PR DESCRIPTION
**Pull request by shargon**
https://github.com/neo-project/neo/pull/1233 in 2x

### CODE REVIEW

File: neo/Consensus/ConsensusService.cs [hunk 1]
```diff
@@ -398,7 +398,7 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
             if (context.RequestSentOrReceived() || context.NotAcceptingPayloadsDueToViewChanging()) return;
             if (payload.ValidatorIndex != context.PrimaryIndex || message.ViewNumber != context.ViewNumber) return;
             Log($"{nameof(OnPrepareRequestReceived)}: height={payload.BlockIndex} view={message.ViewNumber} index={payload.ValidatorIndex} tx={message.TransactionHashes.Length}");
-            if (message.Timestamp <= context.PrevHeader().Timestamp || message.Timestamp > TimeProvider.Current.UtcNow.AddMinutes(10).ToTimestamp())
+            if (message.Timestamp <= context.PrevHeader().Timestamp || message.Timestamp > TimeProvider.Current.UtcNow.AddMinutes(2).ToTimestamp())
```
Comment (lock9) [inline neo/Consensus/ConsensusService.cs]: Quick question: what would happen if the network is actually congested and we need more than 2 minutes to generate a  block?

Comment (shargon) [reply neo/Consensus/ConsensusService.cs]: View change, like now, but not in generate the block, in receive the PrepareRequest, it's different

**Comment by shargon** [comment]
> I think you forgot to run dotnet format. There was the addition of a blank line unnecessarily

Where?

File: neo/Consensus/ConsensusService.cs (hunk 1 above)
Comment (lock9) [inline neo/Consensus/ConsensusService.cs]: Do you want to leave this value hardcoded for Neo 2?

Comment (shargon) [reply neo/Consensus/ConsensusService.cs]: No, I will change it too

**Comment by erikzhang** [comment]
We don't need this on 2.x.

**Pull request closed without merging**